app/services/integrations/aggregator/invoices/payloads/netsuite.rb

```diff
@@ -116,12 +116,22 @@ def item(fee)
             from_property = fee.charge? ? "charges_from_datetime" : "from_datetime"
             to_property = fee.charge? ? "charges_to_datetime" : "to_datetime"
 
+            ns_quantity_limit = 10_000_000_000 # NetSuite quantity max magnitude
```

Comment:
Could you define the max magnitude as a class constant?

```
  NS_QUANTITY_LIMIT = 10_000_000_000 # NetSuite quantity max magnitude
```

Reply:
Shouldn't this be `10_000_000`?